src/quo2/components/list_items/dropdown.cljs

```diff
@@ -0,0 +1,148 @@
+        font-size                                                     (:font-size font)
+        spacing                                                       (case size
+                                                                        :big 4
+                                                                        :medium 2
+                                                                        :small 2)]
+    [rn/touchable-opacity (cond-> {:on-press (fn []
+                                               (if (swap! open? not)
+                                                 (apply-anim dd-height 120)
+                                                 (apply-anim dd-height 0)))
+                                   :style    (merge
+                                              (if icon
+                                                padding-with-icon
+                                                padding-with-no-icon)
+                                              {:width            (if icon
+                                                                   width-with-icon
+                                                                   width)
+                                               :height           height
+                                               :border-radius    (case size
+                                                                   :big 12
+                                                                   :medium 10
+                                                                   :small 8)
+                                               :flex-direction   :row
+                                               :align-items      :center
+                                               :border-width     1
+                                               :border-color     (if @open?
+                                                                   border-color
+                                                                   (color-by-10 border-color))
+                                               :background-color (if @open?
+                                                                   dd-color
+                                                                   (color-by-10 dd-color))})}
+                            disabled? (assoc-in [:style :opacity] 0.3)
+                            disabled? (assoc :disabled true))
+     (when icon
+       [icons/icon icon {:no-color        true
+                         :size            20
+                         :container-style {:margin-right spacing
+                                           :margin-top   1
+                                           :width        icon-size
+                                           :height       icon-size}}])
+     [text/text {:size   font-size
+                 :weight :medium
+                 :font   :font-medium
+                 :color  :main} "Dropdown"]
```

Comment:
I assume this text "Dropdown" should be dynamic.. can you make this a prop and then pass "Dropdown" from the quo2 library?

Reply:
Hey JC, I chatted with Pedro on this .. It's always going to be Dropdown

Reply:
ok cool, then it should be i18n 👍